ouroboros-consensus-test/ouroboros-consensus-test.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version:         3.0
 
 name:                   ouroboros-consensus-test
```

Comment:
The version bounds aren't fixed for `bench-mempool`

Reply:
Done